SecondProgram.cs

```diff
@@ -0,0 +1,191 @@
+
+
+/*
+
+Created By Ragu Balagi Karuppaiah.
+
+Date : 22.06.2019
+
+
+Solution For Second Assignment Second Problem.(Tavisca)
+
+
+
+*/
+
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SecondProblem
+{
+    public class ForumPostEasy

```

Comment:
Create a class in a separate file. You can create a static class with static methods, so that you do not need to create an instance of the class.